tasks/quality_gates.py

```diff
@@ -196,6 +196,7 @@ def parse_and_trigger_gates(ctx, config_path=GATE_CONFIG_PATH):
             gate_states.append({"name": gate, "state": False, "error_type": "AssertionError", "message": str(e)})
         except InfraError as e:
             print(f"Gate {gate} flaked ! (InfraError)\n Restarting the job...")
+            traceback.print_exc()
```

Comment:
Can we instead use 
```suggestion
            traceback.print_exception(e)
```
Doesn't change much in practice perhaps, but it looks safer to not rely on implicit context.

Reply:
Done